easybuild/easyconfigs/p/Perl/Perl-5.22.2-goolf-1.7.20.eb

```diff
@@ -5,16 +5,24 @@ homepage = 'http://www.perl.org/'
 description = """Larry Wall's Practical Extraction and Report Language"""
 
 toolchain = {'name': 'goolf', 'version': '1.7.20'}
-toolchainopts = {'optarch': True, 'pic': True}
+toolchainopts = {'pic': True}
 
 source_urls = ['http://www.cpan.org/src/%(version_major)s.0']
 sources = [SOURCELOWER_TAR_GZ]
 
+# workaround for one extension check
+# be sure you don't build this in NFS folder or you will also get some error
+prebuildopts = 'export PERL_LWP_SSL_VERIFY_HOSTNAME=0 && unset INSTALL_BASE && '
```

Comment:
@boegel one of the perl extensions in my Perl easyconfig required it.

I think this one: https://github.com/hpcugent/easybuild-easyconfigs/blob/5e8d4bdce1bc259e51a1794a21d4a81257c9f6aa/easybuild/easyconfigs/p/Perl/Perl-5.22.2-goolf-1.7.20.eb#L534